sw/airborne/modules/gps/gps_ubx.c

```diff
@@ -194,14 +194,20 @@ static void gps_ubx_parse_nav_pvt(void)
   gps_ubx.state.hacc        = UBX_NAV_PVT_hAcc(gps_ubx.msg_buf) / 10;
   gps_ubx.state.vacc        = UBX_NAV_PVT_vAcc(gps_ubx.msg_buf) / 10;
   gps_ubx.state.sacc        = UBX_NAV_PVT_sAcc(gps_ubx.msg_buf) / 10;
+
+  //FIXME stupid workaround for PVT only
+  gps_ubx.state.pacc = gps_ubx.state.hacc; // report horizontal accuracy
```

Comment:
Doesn't this overwrite a pacc if we read a valid one from another message as well?

Reply:
I don't have a good solution for now. Reconstructing `pacc` from `hacc` and `vacc` is not specified by ublox. With this, it is just not advised to use PVT with an other message providing `pacc`. I would prefer not to add another config flag. Any suggestion ? Using a bool to tell if `pacc` is already updated somewhere else ?